CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # CHANGELOG
 
+## Version 5.0.0, 2024-09-15
```

Comment:
Could we please have the introductory text with the highlights here as well?

Reply:
added